src/compiler/scala/tools/nsc/backend/jvm/GeneratedClassHandler.scala

```diff
@@ -200,7 +200,7 @@ sealed trait CompilationUnitPaths {
 final class CompilationUnitInPostProcess(
   private var classes: List[GeneratedClass],
   val sourceFile: AbstractFile,
-  val outputDir: AbstractFile) extends CompilationUnitPaths with BufferingBackendReporting {
+  val outputDir: AbstractFile) extends CompilationUnitPaths {
```

Comment:
Can we also make `CompilationUnitPaths` a component? https://github.com/scala/scala/commit/48cb0245f495977b6b8db0ee56e468bd5990c48e#diff-1c5772a53859d879a66043aa542d6865R196

Reply:
and ping here